smalltalksrc/VMMakerTests/VMARMV8SIMDEncodingTest.class.st

```diff
@@ -0,0 +1,65 @@
+Class {
```

Comment:
These tests were added to incrementally add SIMD instructions to Pharo, but I'm not sure how valuable it is to keep them, considering that:
- these tests are pretty platform-specific (we disassemble the instructions and check that they're the expected AArch64 SIMD instruction)
- we're also exercising the actual instruction behavior when testing the primitives that use the SIMD instructions

Reply:
Actually... the `DummyProcessor` was needed since the `UnicornProcessor` is not working properly in my Mac with M1. So the discussion on whether to keep it or not should be based on whether we consider it worth it to keep a stubbed processor.